lib/MusicBrainz/Server/Form/Work.pm

```diff
@@ -68,6 +68,35 @@ sub is_empty_attribute {
     return !$field->field('type_id')->value && !$field->field('value')->value;
 }
 
+sub validate_languages {
+    my $self = shift;
+
+    my @languages = $self->field('languages')->fields;
+
+    my $is_valid = 1;
+
+    # If we have only one language, then it can be whatever
+    # If we have two or more, we check it's not mul (248) nor zxx (486)
+    # because combining those with anything else doesn't make sense
+    if (scalar @languages > 1) {
+        for my $language_field (@languages) {
+            if ($language_field->value == 284) {
+                $language_field->push_errors(
+                    l('You can’t select “[Multiple languages]” and specific languages at the same time.')
+                );
+                $is_valid = 0;
+            } elsif ($language_field->value == 486) {
+                $language_field->push_errors(
+                    l('You can’t select “[No lyrics]” and a lyrics language at the same time.')
```

Comment:
Nitpicking: I would probably replace “You can’t” with “You cannot” which is already used for most other strings.